keylime.conf

```diff
@@ -335,7 +338,7 @@ vtpm_policy = {"23":["ffffffffffffffffffffffffffffffffffffffff","000000000000000
 # this file is used if tenant provides "default" as the allowlist file
 # this file should be in the form
 # SHA1sum path_to_file
-allowlist = allowlist.txt
+ima_allowlist = allowlist.txt
```

Comment:
Something seems off with your git merges. This change (and others) are part of already merged changes in keylime.